### PR TITLE
feat: add analytics event to track usage of provider webhooks

### PR DESCRIPTION
**Pull request by davidsoderberg**
### What change does this PR introduce?
Two tracking events so we can track how much provider webhooks is used.

### Why was this change needed?
To be able to understand usage of our provider webhook feature

### CODE REVIEW

File: apps/webhook/src/shared/shared.module.ts [hunk 1]
```diff
@@ -40,6 +41,16 @@ const PROVIDERS = [
     },
   },
   ...DAL_MODELS,
+  {
+    provide: AnalyticsService,
+    useFactory: async () => {
+      const analyticsService = new AnalyticsService(process.env.SEGMENT_TOKEN);
```
Comment (davidsoderberg) [inline apps/webhook/src/shared/shared.module.ts:47]: This needs to be added in terraform I guess? @scopsy

File: apps/webhook/src/webhooks/usecases/webhook/webhook.usecase.ts [hunk 2]
```diff
@@ -34,6 +36,17 @@ export class Webhook {
       channel,
     });
 
+    const member = await this.memberRepository.getOrganizationAdminAccount(command.organizationId);
```
Comment (davidsoderberg) [inline apps/webhook/src/webhooks/usecases/webhook/webhook.usecase.ts:39]: We do not know who created the Webhook so I am using admin account here...

File: apps/webhook/src/webhooks/usecases/webhook/webhook.usecase.ts [hunk 3]
```diff
@@ -44,7 +57,19 @@ export class Webhook {
...
+
+    if (member) {
+      this.analyticsService.track('[Webhook] - Provider Webhook events parsed', member._userId, {
+        _organization: command.organizationId,
+        _environmentId: command.environmentId,
+        providerId,
+        channel,
+        events,
+      });
+    }
```
Comment (scopsy) [inline apps/webhook/src/webhooks/usecases/webhook/webhook.usecase.ts:70]: I think actually that this one will be enough (no need to have the top one. wdyt?

**Review by scopsy — APPROVED** [review]
Looks great! I will add the terraform entry aswell

File: apps/webhook/src/webhooks/usecases/webhook/webhook.usecase.ts (hunk 3 above)
Comment (davidsoderberg) [reply apps/webhook/src/webhooks/usecases/webhook/webhook.usecase.ts:70]: I added both to be able to track if we get more webhooks the we parse...

**Pull request merged**